sequencing/vcf_pca/dxapp.json

```diff
@@ -91,28 +92,28 @@
     },
     {
       "name" : "project_1kg",
-      "label" : "Project onto 1000 Genomes samples",
+      "label" : "Project onto 1000 Genomes samples?",
       "class" : "boolean",
       "optional" : false,
       "default" : true
     },
     {
       "name" : "project_superpop",
-      "label" : "Calculate projection using super-populations (EUR, etc.)",
+      "label" : "Calculate projection using super-populations (EUR, etc.)?",
       "class" : "boolean",
       "optional" : true,
       "default" : false
     },
     {
       "name": "fast_pca",
-      "label": "Use Fast Eigenvalue Approx. (Non fast may not work)",
+      "label": "Use Fast Eigenvalue Approx? (Non fast may not work)",
```

Comment:
"Non fast may not work" doesn't really make sense. Instead, maybe clarify with "Use Fast Eigenvalue Approx? (If false, app may fail.)" or something like that